.github/CONTRIBUTING.md

```diff
@@ -22,8 +22,11 @@ Once you have cloned the project, run `yarn` to install all the dependencies.
 If you encounter trouble in the post-install phase involving `node-sass`, make sure you are building with VC++ 2013: `yarn config set msvs_version 2013`.
 
 #### Architecture :house:
-We have only one peer dependency on [lodash](https://lodash.com/) and we intend to keep it that way.<br />
-immutad●t is organized in namespaces (array, collection, lang, etc.), a lot like lodash, please try to respect this organization; if you are not sure where to put your code, ask for the right place in your issue or PR.
+immutad●t is managed as a [monorepo](https://medium.com/netscape/the-case-for-monorepos-907c1361708a) using [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) and [lerna](https://lernajs.io/)
+
+The main package has no peer dependency and we intend to keep it that way.
```

Comment:
Since I had to add a dependency on `babel-runtime`, we should either change this sentence or remove it.